std_execution.bs

```diff
@@ -3205,23 +3205,23 @@ enum class forward_progress_guarantee {
 1. A <i>receiver</i> represents the continuation of an asynchronous operation. An asynchronous operation may complete with a (possibly empty) set of values, an error, or it may be cancelled. A receiver has three principal operations corresponding to the three ways
     an asynchronous operation may complete: `set_value`, `set_error`, and `set_done`. These are collectively known as a receiver’s <i>completion-signal operations</i>.
 
-2. The `receiver` concept defines the requirements for a receiver type with an unknown set of value types. The `receiver_of` concept defines the requirements for a receiver type with a known set of value types, whose error type is `std::exception_ptr`.
+2. The `receiver` concept defines the requirements for a receiver type with an unknown set of value types. The `receiver_of` concept defines the requirements for a receiver type with a known set of value types, whose error type is `exception_ptr`.
 
     <pre highlight=c++>
     template&lt;class T, class E = exception_ptr>
     concept receiver =
       move_constructible&lt;remove_cvref_t&lt;T>> &&
       constructible_from&lt;remove_cvref_t&lt;T>, T> &&
       requires(remove_cvref_t&lt;T>&& t, E&& e) {
-        { execution::set_done(std::move(t)) } noexcept;
-        { execution::set_error(std::move(t), (E&&) e) } noexcept;
+        { execution::set_done(move(t)) } noexcept;
+        { execution::set_error(move(t), (E&&) e) } noexcept;
```

Comment:
Please leave `std::` for `std::move` and `std::forward`. The standard does that for historical reasons, I'm told.